Resources/Storage/GFAL2_StorageBase.py

```diff
@@ -710,6 +713,9 @@ def __getSingleMetadata( self, path ):
           metadataDict['Unavailable'] = 1
         else:
           metadataDict['Unavailable'] = 0
+        if attributeDict['user.status'] == 'ONLINE_AND_NEARLINE':
```

Comment:
yes, but until I have time to unify, please hardcode in xroot. Or just hardcode it in the base, and have it overwriten by srm